static/mainapp/css/layout.css

```diff
@@ -45,11 +45,11 @@ body {
 }
 
 #bookName {
-  border-radius: 10px;
+  border-radius: 50px;
 }
 
 #searchdropdown {
-  margin-left: 50%;
+  margin-left: 57%;
```

Comment:
let it be 50%

Reply:
But @maneprajakta when I signed in my margin looked fine.

![Annotation 2021-04-04 111822](https://user-images.githubusercontent.com/56751963/113499931-9d50f300-9537-11eb-910e-a9bffde1c47d.png) 

Let's see what others say! and if its a problem for others, then surely I would keep it as default (50%)